packages/flutter_tools/templates/ffi_plugin/.gitignore.tmpl

```diff
@@ -0,0 +1,29 @@
+# Miscellaneous
+*.class
+*.log
+*.pyc
+*.swp
+.DS_Store
+.atom/
+.buildlog/
+.history
+.svn/
+
+# IntelliJ related
+*.iml
+*.ipr
+*.iws
+.idea/
+
+# The .vscode folder contains launch configuration and tasks you configure in
+# VS Code which you may wish to be included in version control, so this line
+# is commented out by default.
+#.vscode/
+
+# Flutter/Dart/Pub related
+# Libraries should not include pubspec.lock, per https://dart.dev/guides/libraries/private-files#pubspeclock.
+/pubspec.lock
+**/doc/api/
+.dart_tool/
+.packages
+build/
```

Comment:
Can't we share most or all of this file with the other plugin template?

Reply:
The platform channel plugin and FFI plugin are separate templates. So they have a separate sub folder in the templates folder. I think it would make the code less comprehensible trying to share some files.

Reply:
We already share common code for the two different app templates; why is doing so with two plugin templates more confusing?